pkg/provisioner/resourcehandlers.go

```diff
@@ -235,7 +235,7 @@ func deleteExistingSecretAndConfigMapIfExist(obc *v1alpha1.ObjectBucketClaim, c
 			return fmt.Errorf("found OBC credentials Secret %q not owned by OBC, assuming this is a user-created resource, user must delete the Secret in order to allow OBC provisioning to continue", secret.Name)
```

Comment:
On [L230](https://github.com/kube-object-storage/lib-bucket-provisioner/pull/204/files#diff-ab88f1a4bb2c3e7d95d10e30594e543aa8649561fde3173455aa2f960f24e967R230) do you want to handle IsNotFound error?

Reply:
The `getExistingSecret` function handles the `IsNotFound` error by returning a nil secret and nil error.